docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -62,9 +62,6 @@ Property Name                                      Description
                                                    ``EACH_QUORUM``, ``QUORUM``, ``LOCAL_QUORUM``, ``ONE``, ``TWO``,
                                                    ``THREE``, ``LOCAL_ONE``, ``ANY``, ``SERIAL``, ``LOCAL_SERIAL``.
 
-``cassandra.allow-drop-table``                     Set to ``true`` to allow dropping Cassandra tables from Trino
```

Comment:
I think there is some confusion. Only JDBC based connectors will be affected by what @electrum mentioned about `allow-drop-table`. So Cassandra would still need this config most probably.

Also, let's not remove things from documentation until the feature actually gets implemented.